src/UI/EmojiSupport.cpp

```diff
@@ -60,7 +60,7 @@ TMPro::TMP_SpriteAsset* CreateTMP_SpriteAsset() {
     texture->SetPixels(clearPixels);
     texture->Apply(false, true);
 
-    TMPro::TMP_SpriteAsset* spriteAsset = TMPro::TMP_SpriteAsset::New_ctor();
+    TMPro::TMP_SpriteAsset* spriteAsset = ScriptableObject::CreateInstance<TMPro::TMP_SpriteAsset*>();
```

Comment:
This change seems out of place 🤔

Reply:
This fixed a runtime warning that is produced, when the mod runs. The warning is visible in logcat and looks something like this:
"TMPro.TMP_SpriteAsset must be instantiated using the ScriptableObject.CreateInstance method instead of new TMP_SpriteAsset" (dont have the log here right now but that is what i googled back then)

According to this description by Resharper this is needed because otherwise Unity doesnt get access to the Start and Update Method of the new Object:
https://github.com/JetBrains/resharper-unity/wiki/ScriptableObjects-must-be-instantiated-with-ScriptableObject.CreateInstance-instead-of-new